stream_alert_cli/terraform_generate.py

```diff
@@ -237,14 +240,19 @@ def generate_stream_alert(cluster_name, cluster_dict, config):
                     tf_key: modules['stream_alert']['alert_processor']['outputs'][output]
                 })
 
-    # Add Rule Processor input config
+    # Add Rule Processor input config from the loaded cluster file
     input_config = modules['stream_alert']['rule_processor'].get('inputs')
     if input_config:
-        cluster_dict['module']['stream_alert_{}'.format(cluster_name)].update({
-            'input_sns_topics': input_config['aws-sns']
-        })
+        input_mapping = {
+            'input_sns_topics': 'aws-sns'
+        }
+        for tf_key, input_key in input_mapping.iteritems():
```

Comment:
Why the `for` loop when there is only one item? Is there chance that this will be expanded in the future with more keys?